application/templates/static_site/error/README.md

```diff
@@ -0,0 +1,12 @@
+## Static site error pages
+
+This directory structure (`static_site/error`) is depended upon by CloudFront and Heroku. Heroku serves its static 
```

Comment:
For clarity, maybe we should rename these folders so that we have:

> external_error_pages/publisher/ –  ones used by Heroku
> external_error_pages/website/ - ones used by Cloudfront

Reply:
I feel like doing that will change the scope of this ticket, really. This is just about updating the error pages to be styled consistently, whereas changing the structure needed by CloudFront/Heroku is a piece of work that involves (I think) a more thoughtful release plan and consideration. Going to leave this as-is for now.